selfdrive/car/tests/test_docs.py

```diff
@@ -74,8 +74,8 @@ def test_harnesses(self):
         if car.name == "comma body":
           raise unittest.SkipTest
 
-        self.assertNotIn(car.harness_kit.connector, [None, Harness.none], f"Need to specify car harness: {car.name}")
-        self.assertTrue(car.harness_kit.parts, f"Need to specify harness parts: {car.name} with {car.harness_kit.connector.value} connector")
+        self.assertTrue(car.car_parts.parts, f"Need to specify car parts: {car.name}")
```

Comment:
can you think of any more good tests? like you need at least one mount type, etc.

Reply:
The only overlapping prerequisite for all cars is that they need a mount, a connector and some cables. We currently test for all three. Do we want to enforce anything else? We could enforce stuff like a single connector, but that might not be true forever.